File: src/main/java/com/squarepolka/digiscope/plot/recording/HorizontalGraph.java
package com.squarepolka.digiscope.plot.recording;

import java.math.BigDecimal;

public class HorizontalGraph {

    private BigDecimal maxVoltage;
    private BigDecimal minVolgate;
    private BigDecimal maxTime;

    public HorizontalGraph(BigDecimal maxVoltage, BigDecimal minVoltage, BigDecimal maxTime) {
        this.maxVoltage = maxVoltage;
        this.minVolgate = minVoltage;
        this.maxTime = maxTime;
    }

}

File: src/main/kotlin/graph/HorizontalGraph.kt
package graph

import java.math.BigDecimal

class HorizontalGraph (maxVoltage: BigDecimal, minVoltage: BigDecimal, maxTime: BigDecimal) {

    private val maxVoltage: BigDecimal
    private val minVoltage: BigDecimal
    private val maxTime: BigDecimal

    init {
        this.maxVoltage = maxVoltage
        this.minVoltage = minVoltage
        this.maxTime = maxTime
    }

    fun test() {

        println("hello ")
    }
}